sim/src/kernels/common.opencl

```diff
@@ -28,6 +28,11 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+/**
+  * Define NVIDIA pragma for assuming a 32-wide SIMD
+  */
+#define NVIDIA 1
```

Comment:
This should really be programatically set ((or detected) based on the platform being used